src/fmeval/transforms/common.py

```diff
@@ -14,7 +14,13 @@ class GeneratePrompt(Transform):
     applying the same prompt template to each input.
     """
 
-    def __init__(self, input_keys: List[str], output_keys: List[str], prompt_template: str):
+    def __init__(
+        self,
+        input_keys: List[str],
+        output_keys: List[str],
+        prompt_template: str,
+        placeholder_keys_dict: Optional[Dict[str, str]] = None,
```

Comment:
Maybe rename this to something like `placeholder_to_record_key`, since the mapping is from a placeholder (like `"question"`) to a record key, where said record key maps to the value to be substituted in for the placeholder.